bundle/edu.gemini.pot/src/test/scala/edu/gemini/spModel/target/env/GuideGroupSpec.scala

```diff
@@ -35,19 +33,19 @@ class GuideGroupSpec extends Specification with ScalaCheck with Arbitraries {
         g.grp match {
           case a: AutomaticGroup => g.setName(nopt) == g
           case _                 => g.setName(nopt).getName == nopt &&
-                                    g.setName(n).getName == nopt
+            g.setName(n).getName == nopt
```

Comment:
Trivial, but I guess if we don't line these up then let's place the whole thing (for both branches) on a second line:

````scala
case a: AutomaticGroup =>
  g.setName(nopt) == g
case _                 =>
  g.setName(nopt).getName == nopt && g.setName(n).getName == nopt
````